spec/services/exotel_api_service_spec.rb

```diff
@@ -0,0 +1,78 @@
+require 'rails_helper'
+
+describe ExotelAPIService, type: :model do
+  describe '#call_details' do
+    let!(:call_details_200) { File.read('spec/support/fixtures/call_details_200.json') }
```

Comment:
We tried VCR here but Exotel exposes the token in the response hence we'd have to mock out the VCR response manually. Instead of doing that, we avoided using a VCR mock entirely and made a custom mock instead.